MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -255,7 +291,8 @@ private void OnPlayingChanged(object sender, PlayingChangedEventArgs e)
         {
             if (sender != CurrentPlaybackManager) return;
             MediaFinished?.Invoke(sender, e);
-            await PlayNext();
+            if (MediaQueue.Repeat == RepeatType.RepeatOne) await Seek(TimeSpan.Zero);
```

Comment:
But then i would go into the `SetNextAsCurrent()` from the Queue and not to the `PlayNext()`